src/instance.ts

```diff
@@ -106,10 +127,16 @@ Please use the format 'my-instance' or '${bigtable.projectName}/instances/my-ins
    * @param {string} name The name of the app profile.
    * @returns {AppProfile}
    */
-  appProfile(name) {
+  appProfile(name: string) {
     return new AppProfile(this, name);
   }
 
+  create(options: CreateInstanceOptions): Promise<CreateInstanceResponse>;
+  create(callback: CreateInstanceCallback): void;
```

Comment:
It looks like `options: CreateInstanceOptions` is not an optional param (due to `Clusters`);
This Overload should be removed